### ISSUE TITLE
Report images appear to be uncompressed

### ISSUE THREAD

**Issue by ariekahn** [poldracklab/fmriprep#2159]
fMRIPrep 20.1.0

I'm just observing that the svg files generated in the fMRIPrep report are significantly larger than necessary. If you take, for instance, one of the `*bold.svg* images and compress it, you get around a two-fold reduction in size:
```
 > ls -lh sub-GLS003_ses-2_task-graphlocalizer_desc-bbregister_bold.svg
2.9M May 30 13:21 sub-GLS003_ses-2_task-graphlocalizer_desc-bbregister_bold.svg
 > gzip sub-GLS003_ses-2_task-graphlocalizer_desc-bbregister_bold.svg
 > ls -lh sub-GLS003_ses-2_task-graphlocalizer_desc-bbregister_bold.svg.gz
1.2M May 30 13:21 sub-GLS003_ses-2_task-graphlocalizer_desc-bbregister_bold.svg.gz
```

Obviously this isn't a solution at all, but I'm throwing this out there in case there's a better option for handling the image data. I think this might, in particular, come down to how SVG files handle embedded raster data, as a brief google search suggests they typically don't compress it at all. 

This applies to all the embedded brain images, which means that the svg files end up being larger than the `boldref.nii.gz` volumes (in this case, a similar boldref image is 628kb). With my own dataset, the image directory for each subject ends up being over 100MB, so this ends up being a non-negligible amount (as well as very slow loading when opening `report.html` over a networked drive.)

**Comment by ariekahn** [comment]
I noticed that there appears to be ongoing work with this in niworkflows, so closing this.

**Comment by oesteban** [comment]
I don't think we ever considered just gzipping the svgs. That is definitely an easy thing to do, if browsers can easily use compressed svgs.

**Comment by ariekahn** [comment]
Can they? I was really just doing that to gauge what sort of filesize difference there would be, not as an implementation suggestion.

**Comment by oesteban** [comment]
> Can they?

I don't know either, but before closing this issue I think it'd be nice we checked.

**Comment by ariekahn** [comment]
There _is_ a native compressed variant, `svgz`, but from what I can tell, it seems like there's pretty spotty support for local viewing without an http server.

**Comment by oesteban** [comment]
https://www.w3.org/TR/SVG11/intro.html#MIMEType
